grunz/tests/test_file_utils.py

```diff
@@ -0,0 +1,156 @@
+import unittest
+import tempfile
+import os
+import time
+import hashlib
+from pathlib import Path
+from filecmp import cmp
+
+from grunz.file_utils.file_utils import FileUtils
+
+
+class TestFileUtils(unittest.TestCase):
+
+    def setUp(self):
```

Comment:
Nice you're using setup and teardown methods here Chris. They'll ensure proper isolation and efficent resource management! :)

Reply:
Good to hear, thanks!